addons/io_scene_gltf2/__init__.py

```diff
@@ -384,6 +396,8 @@ def execute(self, context):
         export_settings['gltf_extras'] = self.export_extras
         export_settings['gltf_yup'] = self.export_yup
         export_settings['gltf_apply'] = self.export_apply
+        export_settings['export_loose_edges'] = self.export_loose_edges
+        export_settings['export_loose_vertices'] = self.export_loose_vertices
```

Comment:
Let's name these export options `use_mesh_edges` and `use_mesh_vertices`. The first will match the FBX addon, so we have a more consistent scripting API. The OBJ exporting uses `use_edges` instead, which so we can't be 100% consistent, but at least we avoid introducing a third name.

Reply:
Sounds good, I will give it a crack sometime this week.